conanfile.py

```diff
@@ -15,15 +15,17 @@ class MeshlinklibConan(ConanFile):
 
     def build(self):
         autotools = AutoToolsBuildEnvironment(self)
-        self.run("cd .. && autoreconf -fsi ")
-        autotools.configure(configure_dir="..",args=[ "--prefix=${PWD}" ])
+        self.run("cd .. && autoreconf -fsi")
+        autotools.configure(configure_dir="..", args=[ "--prefix=${PWD}" ])
+        # This is a temporary fix for the error - "Error 512 while executing make -j1". Once the issue is resolved in the meshlink's build process, this will be removed.
         self.run("cd ../doc && sed -e s,'@PACKAGE\@',\"meshlink\",g -e s,'@VERSION\@',\"0.1\",g -e s,'@sysconfdir\@',\"/usr/local/etc\",g -e s,'@localstatedir\@',\"/usr/local/var\",g include.texi.in > include.texi")
```

Comment:
Split into multiple line if line length is greater than 104 characters

Reply:
handled